src/main/java/com/peachbros/letsmerge/mission/model/domain/Mission.java

```diff
@@ -0,0 +1,32 @@
+package com.peachbros.letsmerge.mission.model.domain;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+public class Mission {
+    public static final String MISSION_DATE_FORMAT = "yyyy-MM-dd HH:mm";
```

Comment:
```suggestion
    public static final DateTimeFormatter MISSION_DATE_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
```

이렇게 해도 되지 않을까요?
다른 곳에서도 많이 쓰이는 것 같은데 매번 객체를 생성해 주는 것 보다는 하나 만들어서 재사용 하는게 나은 것 같아요

Reply:
아이고 좋은 생각입니다! :)